docs/_data/_routes.yml

```diff
@@ -24,8 +24,8 @@ platforms:
         icon: /assets/images/shield.svg
         description: Configure rules, settings, and limits for your company’s spending.
 
-      - href: expenses
-        title: Expenses
+      - href: expenses-&-payments
+        title: Expenses & Payments
```

Comment:
@strepanier03 the below changes are nicely done!

But we need to fix this file. You've renamed the articles for new expensify, but in `_routes` file, expensify classic was renamed. So you'll have to undo this change in _routes and make the change for new expensify in this file.
It's at the end of the file.

Let me know if you have any questions : )

Reply:
Thank you Rushat, I will work on fixing this now.